src/convert/mapEvent.js

```diff
@@ -39,12 +50,15 @@ const mapEvent = (event) => {
   const keys = event.changes.keys;
   const changes = Array.from(keys.entries(), convertMapOp);
 
-  const baseOp = {
-    type: 'set_node',
-    newProperties: {},
-    properties: {},
-    path: toSlatePath(event.path),
-  };
+  for( var [key, action] of keys.entries()){}
```

Comment:
I don't understand this line?

Reply:
I used this maneuver to get the key and actions of the 'event.changes.keys' to be used inside the expression below, turned out that if there were multiples fields updated this logic won't work. I changed to check if the data field exists inside the keys object If so I check then if the action is 'update' and if the path points to top-level or inside a node.